README.md

```diff
@@ -42,6 +42,18 @@ Project: Messenger with translating(Bachelor work)
  - Також було дописано Unit-тести для нового функціоналу. Зараз проект покрито десь на 65-70%, що досить добре.
  - Також було відформатовано і відредаговано деякі частини коду.
 
+ # Messanger v2.0 БД
```

Comment:
where is task#6?

Reply:
I haven't done task 6 yet. I asked you if I could pass task 7 first. And you let me.